src/components/Person/Person.jsx

```diff
@@ -1 +1,16 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
+  <div>
+    <h2 className="Person__name">{`My name is ${person.name}`}</h2>
+
+    {person.age && <p className="Person__age">{`My name is ${person.age}`}</p>}
+
+    <p className="Person__partner">
+      {person.sex === 'm' && person.isMarried
+        ? `My wife is ${person.partnerName}`
+        : ''}
+      {person.sex === 'f' && person.isMarried
+        ? `My husband is ${person.partnerName}`
```

Comment:
decide before jsx what text will be rendered and just render in jsx. something like that

```
const personPartnerType = sex === 'm' ? 'wife' : 'husband';
const personPartner = isMarried ? `${partnerName} is my ${personPartnerType}` : 'I am not married'
```